tests/test_cli.py

```diff
@@ -873,3 +873,12 @@ def test_edit_raw(todo_factory, runner):
 
     assert not result.exception
     assert not result.output
+
+
+def test_new_description_from_stdin(runner):
+    result = runner.invoke(cli, ['new', '-l', 'default', '-r', 'hello'],
+                           input='world\n')
+    assert not result.exception
```

Comment:
I believe it's also important to check that the word `world` ends up in the right field in the created todo. Probably can be done following the pattern [here](https://github.com/pimutils/todoman/blob/05b7f915be7b04fb16a0c1fb5940d5f504cc7add/tests/test_cli.py#L709-L710).